README.md

```diff
@@ -30,66 +30,259 @@ From the GNU site:
+
+    #Scrape pastebin for onions (needs paid account / IP whitelisting)
+    */5 * * * * /home/freshonions-torscraper/scripts/pastebin.sh
+
+    #Portscan new onions
+    1 13 * * * /home/freshonions-torscraper/scripts/portscan_up.sh
+
+    #Scrape stronghold paste
+    32 */2 * * * /home/freshonions-torscraper/scripts/stronghold_paste_rip.sh
+
+    #Detect clones
+    20 14 * * * /home/freshonions-torscraper/scripts/detect_clones.sh
+
+    #Keep a sql dump of data
+    1 */1 * * * mysqldump -u username -ppassword --database tor --result-file=/home/dump.sql
+    1 */8 * * * mysqldump -u username -ppassword --database tor --result-file=/home/dump_backup.sql
 
 
 ## Infrastructure
 
-Fresh Onions runs on two servers, a frontend host running the database and hidden service web site, and a backend host running the crawler. Probably most interesting to the reader is the setup for the backend. TOR as a client is COMPLETELY SINGLETHREADED. I know! It's 2017, and along with a complete lack of flying cars, TOR runs in a single thread. What this means is that if you try to run a crawler on a single TOR instance you will quickly find you are maxing out your CPU at 100%.
+Fresh Onions runs on two servers, a frontend host running the database and hidden service website, and a backend host running the crawler. Probably most interesting to the reader is the setup for the backend. T
+OR as a client is COMPLETELY SINGLETHREADED. I know! It's 2017, and along with a complete lack of flying cars, TOR runs in a single thread. What this means is that if you try to run a crawler on a single TOR instance you will quickly find you are maxing out your CPU at 100%.
 
 The solution to this problem is running multiple TOR instances and connecting to them through some kind of frontend that will round-robin your requests. The Fresh Onions crawler runs eight Tor instances.
 
-Debian (and ubuntu) comes with a useful program "tor-instance-create" for quickly creating multiple instances of TOR. I used Squid as my frontend proxy, but unfortunately it can't connect to SOCKS directly, so I used "privoxy" as an intermediate proxy. You will need one privoxy instance for every TOR instance. There is a script in "scripts/create_privoxy.sh" to help with creating privoxy instances on debian systems. It also helps to replace /etc/privoxy/default.filter with an empty file, to reduce CPU load by removing unnecessary regexes.
+Debian (and Ubuntu) comes with a useful program "tor-instance-create" for quickly creating multiple instances of TOR. I used Squid as my frontend proxy, but unfortunately, it can't connect to SOCKS directly, so I
+ used "Privoxy" as an intermediate proxy. You will need one Privoxy instance for every TOR instance. There is a script in "scripts/create_privoxy.sh" to help with creating Privoxy instances on Debian systems. It also helps to replace /etc/privoxy/default.filter with an empty file, to reduce CPU load by removing unnecessary regexes.
```

Comment:
whitespace changes warning!

It looks like a newline was added in TOR (after the T) if this is the only change, it should be reverted. Whitespace changes contribute to nothing except git diff noise.